fearless/Modules/Staking/StakingMain/StakingMainInteractor+InputProtocol.swift

```diff
@@ -18,6 +18,7 @@ extension StakingMainInteractor: StakingMainInteractorInputProtocol {
         provideRewardCalculator()
         provideEraStakersInfo()
         provideNetworkStakingInfo()
+        fetchEraCompletionTime()
```

Comment:
We should find home for this call. 

The problem here is that we could receive this info in some time in the past and at some point in future the state of NominatorView changes and we are starting displaying incorrect timer. Probably we should store local time when the value was fetched and then we can always correct this value by providing current local time. 

Also we should request new value when era changes. I think we can pair this request with `provideEraStakersInfo` call because the interactor is notified when this happens and basically this means that era changed.